src/ring_theory/ideal/operations.lean

```diff
@@ -1235,6 +1235,12 @@ by simp only [ring_hom.algebra_map_to_algebra, ring_hom.comp_id]
 instance {I : ideal A} : is_scalar_tower R A (ideal.quotient I) :=
 is_scalar_tower.of_algebra_map_eq' (quotient.alg_map_eq R I)
 
+instance quotient.is_scalar_tower' [algebra S A] [algebra S R] [is_scalar_tower S R A]
```

Comment:
I think this one subsumes the instance above, so the one above can be removed. I doubt that will fix the timeout though.

Thanks for making the PR

Reply:
Indeed it doesn't...

If we want to keep this just for the record, should I close the PR?

Reply:
I'll have a play with this tomorrow, now that you have a cache ready for me!